.github/workflows/kfp-samples.yml

```diff
@@ -35,11 +35,13 @@ jobs:
       uses: ./.github/actions/kfp-cluster
       with:
         k8s_version: ${{ matrix.k8s_version }}
+      continue-on-error: true
 
     - name: Build and upload the sample Modelcar image to Kind
       run: |
         docker build -f samples/v2/modelcar/Dockerfile -t registry.domain.local/modelcar:test .
         kind --name kfp load docker-image registry.domain.local/modelcar:test
+      continue-on-error: true
```

Comment:
Won't this cause `Run Samples Tests` to run even though the environment failed to get setup?

Reply:
It will. Seems like we need to revise all workflows that collect logs.
For now, let's focus on this one.
I pushed a commit with the fix and forced a failure. Let's see what happens.

Reply:
It works as expected now.
https://github.com/kubeflow/pipelines/actions/runs/14092383167/job/39472012954?pr=11773
I reverted the forcing failure.